### PR TITLE
Create Interface

### LINKED ISSUE #1: Interface
## Create an interface for the game with 
- the name of the game in the center
-  a time counter in the upper right corner

### PR DESCRIPTION
**Pull request by ArtemFiend**
## Whats added:
I create a main ```interface``` with name of the game

## Issue:
This pull requests close #1

### CODE REVIEW

File: interface/interface.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+def application():
+    app = QApplication(sys.argv)
+    window = QMainWindow()
+
+    window.setWindowTitle('Citadel')
+    window.setGeometry(400, 250, 500, 300)
+
+    main_text = QtWidgets.QLabel(window) #текст в центре
+    main_text.setText('Citadel') #текст в центре
+    main_text.move(225, 125) #текст в центре
```
Comment (Arrgentum) [inline interface/interface.py]: please rewrite the comments in English

File: interface/interface.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
+from PyQt6 import QtWidgets
+from PyQt6.QtWidgets import QApplication, QMainWindow
```
Comment (Arrgentum) [inline interface/interface.py]: for my opinion, these imports are somewhat similar, they cannot be combined?

File: interface/interface.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+import sys
+
+def application():
+    app = QApplication(sys.argv)
+    window = QMainWindow()
+
+    window.setWindowTitle('Citadel')
+    window.setGeometry(400, 250, 500, 300)
```
Comment (Arrgentum) [inline interface/interface.py]: please enter constants

File: interface/interface.py (hunk 1 above)
Comment (ArtemFiend) [reply interface/interface.py]: done that

File: interface/interface.py (hunk 2 above)
Comment (ArtemFiend) [reply interface/interface.py]: the application will not work

File: interface/interface.py (hunk 3 above)
Comment (Arrgentum) [reply interface/interface.py]: rewrite it depending on the size of the computer screen

**Review by Arrgentum — APPROVED** [review]

**Pull request merged**